server/src/main/scala/za/co/absa/atum/web/api/controller/CheckPointController.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.web.api.controller
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.{HttpStatus, ResponseEntity}
+import org.springframework.web.bind.annotation.{PostMapping, RequestBody, RestController}
+import za.co.absa.atum.model.dto.CheckpointDTO
+import za.co.absa.atum.web.api.service.CheckPointService
+
+
+@RestController
+class CheckPointController {
+
+  @Autowired
+  private val checkpointService: CheckPointService = null
```

Comment:
The IntelliJ says: `Could not autowire. No beans of 'CheckPointService' type found.`

Reply:
I am working on it, but I wouldn't know because it doesn't immerge on my side.

Reply:
I have pushed some changes, please test again to if the issues are resolved.